src/common/buildConfigEntityAttributes.ts

```diff
@@ -93,6 +93,7 @@ export const buildConfigEntityAttributes = {
   brewPullActive: {
     id: 'brewPullActive',
     title: 'Brew Pull Active',
+    tooltip: 'Option enables to search for built dependencies also in brew not just in pnc.',
```

Comment:
`The option enables to search for built dependencies in Brew as well as in PNC.`